include/wx/cmndata.h

```diff
@@ -145,6 +145,24 @@ class WXDLLIMPEXP_CORE wxPrintData: public wxObject
     wxDECLARE_DYNAMIC_CLASS(wxPrintData);
 };
 
+/*
+ * wxPrintPageRange
+ * Contains a range of pages to be printed.
+ */
+
+class WXDLLIMPEXP_CORE wxPrintPageRange
+{
+public:
+    int fromPage = 0;
+    int toPage = 0;
+
+    wxPrintPageRange() : fromPage(0), toPage(0) { }
```

Comment:
This is definitely not needed, we should just use
```suggestion
    wxPrintPageRange() = default;
```